### ISSUE TITLE
Build fails on ubuntu:noble in Docker, ../src/fft.h:95:25: error: expected ')' before 'error'

### ISSUE THREAD

**Issue by sblandford** [PhantomSDR/PhantomSDR#2]
I am trying to compile PhantomSDR as a Docker using the latest Ubuntu release.

ninja fails with a compilation error:
```
53.47 In file included from ../src/fft.cpp:1:
53.47 ../src/fft.h:95:25: error: expected ')' before 'error'
53.47    95 |     errorFFT(std::string error) : noFFT(0, 0) { throw error; }
53.47       |             ~           ^~~~~~
53.47       |                         )
```

This can be reproduced by building the attached zipped Dockerfile.

[Dockerfile.zip](https://github.com/PhantomSDR/PhantomSDR/files/14432235/Dockerfile.zip)

**Comment by rhgndf** [comment]
Fixed in 982bab8. The code is removed now.

**Comment by sblandford** [comment]
Compiles OK now, thanks.
